ArduPlane/radio.cpp

```diff
@@ -402,6 +399,10 @@ bool Plane::rc_throttle_value_ok(void) const
  */
 bool Plane::rc_failsafe_active(void) const
 {
+    if ( ThrFailsafe(g.throttle_fs_enabled.get()) == ThrFailsafe::Disabled) {
```

Comment:
I think this means that we will start using invalid RC inputs if the fail safe is disabled.

Reply:
I should move it to the bottom of this method....will do